external-crates/move/crates/bytecode-verifier-tests/src/unit_tests/bounds_tests.rs

```diff
@@ -222,16 +222,14 @@ fn invalid_signature_for_vector_operation() {
 
     let skeleton = basic_test_module();
     let sig_index = SignatureIndex(skeleton.signatures.len() as u16);
-    for bytecode in vec![
-        VecPack(sig_index, 0),
+    for bytecode in [VecPack(sig_index, 0),
```

Comment:
ick is this what cargo fmt does for this? 🤢

Reply:
Looks like it :(

Reply:
Actually it doesn't. Our rustfmt is really fucked for external-crates